activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,46 @@
+require 'active_support/time'
+
+module ActiveSupport  
+  class Claims
+    class InvalidClaims < StandardError; end
+    class ExpiredClaims < StandardError; end
+    
+    attr_reader :payload, :purpose, :expires
+
+    def initialize(options)
```

Comment:
Because they're tough to pass to `pick_purpose` and `pick_expiration`. I'd still like `value` as a kw arg though.